src/main.py

```diff
@@ -135,6 +135,9 @@ async def handle_event(event: Dict, event_context, projects_ids: Optional[List[s
         disabled_apis = {}
         for project_id in projects_ids:
             disabled_apis = {project_id: await get_all_disabled_apis(context, token, project_id)}
+            if disabled_apis[project_id] == {project_id}:
+                projects_ids.remove(project_id)
```

Comment:
I'm afraid we can't remove project_id during iteration - I'm not sure what Python does in this case... as I mentioned in the comment about logging, I would collect all projects with disabled monitoring api and after that remove them from projects_ids and log